src/js/components/homepage/features/PaneFeature.jsx

```diff
@@ -13,27 +14,44 @@ const PaneFeature = () => (
                 <div className="feature-pane__content">
                     <h3 className="feature-pane__content-title">New! - Congressional Budget Justifications</h3>
                     <p className="feature-pane_content-text">See the rich, detailed documents each agency submits to Congress to justify its annual budget request.</p>
-                    <div className="feature-pane__button-wrapper">
+                    <button className="feature-pane__button-wrapper">
                         <a href="/#/agency" className="feature-pane__button">Explore Budget Justifications</a>
-                    </div>
+                    </button>
                 </div>
                 <div className="feature-pane__content-divider" />
                 <hr className="feature-pane__mobile-content-divider" />
-                <div className="feature-pane__content">
-                    <h3 className="feature-pane__content-title">New! – Your Guide to America’s Finances</h3>
-                    <p className="feature-pane_content-text">Your Guide presents straightforward information about the federal government’s spending and revenue, as well as the deficit and debt in 2018.</p>
-                    <div className="feature-pane__button-wrapper">
-                        <a href="https://datalab.usaspending.gov/americas-finance-guide/" target="_blank" rel="noopener noreferrer" className="feature-pane__button">Explore the Data</a>
+                <div className="feature-pane__content feature-pane__content-middle__sub">
+                    <div className="">
+                        <div id="feature-pane__content-header__middle">
+                            Updated for FY 2019
```

Comment:
looks like this should be `feature-pane__content-middle__sub`

Reply:
my bad, this comment should be on line 24

Reply:
Cool, so, if there's no className, we should just not even use the className attribute.

Reply:
just pushed